oca_port/migrate_addon.py

```diff
@@ -240,3 +263,11 @@ def _print_tips(self, blacklisted=False):
             new_pr_url=new_pr_url,
         )
         print(tips)
+
+    def _apply_code_pattern(self):
+        print("Apply code pattern...")
+        subprocess.run(
+            f"odoo-module-migrate -i {self.app.source_version} -t {self.app.target_version} --modules {self.app.addon} --directory {self.app.repo_path} ",
```

Comment:
To be compatible with modules not only located at the root folder, I needed to change this:
```suggestion
            f"odoo-module-migrate -i {self.app.source_version} -t {self.app.target_version} --modules {self.app.addon} --directory {self.app.addons_rootdir} ",
```

`self.app.addons_rootdir` = `odoo/local-src` if your module is located in `odoo/local-src/my_module/`. E.g the command I'm using in a private project:

```
$ oca-port origin/master origin/18.0 --source-version=14.0 odoo/local-src/my_module --upstream-org=camptocamp --verbose
```

Otherwise it is working really fine :+1:

Reply:
Maybe another question: could we use instead the API of `odoo-module-migrator`, by  using directly the `Migration` class like it's done by the CLI here: https://github.com/OCA/odoo-module-migrator/blob/master/odoo_module_migrate/__main__.py#L143-L156

So we avoid a call to subprocess.

Reply:
Hi @sebalix, it's actually a good idea, I updated the code

I spare the option **Skip removing migration folder** as I am not sure how to best implement it, thinking of adding new attribute `skip_remove_migration_folder` in object App, how does it sound to you?

Reply:
Let's keep the defaults of OCA: we want to remove them. As it's a change commited, user can still revert it in case he wants to keep it. But it's an exception, we often want to remove them anyway.

Reply:
totally agree, so current state is ready for review